server/sonar-server/src/main/java/org/sonar/server/component/ws/ProjectsWs.java

```diff
@@ -25,6 +25,8 @@
 import org.sonar.api.server.ws.WebService;
 
 public class ProjectsWs implements WebService {
+  public static final String ENDPOINT = "api/projects";
```

Comment:
I see it is used in unit test. Code shouldn't be altered for the sole benefit of unit tests. Use a separate constant in the unit test.
Additional benefit is that is someone or something (refactoring) changes the endpoint string in the code, the change will be detected by the unit test